main.tf

```diff
@@ -115,6 +116,20 @@ resource "aws_instance" "web-server-test" {
   }
 }
 
+#creating EBS resources (more RAM) and attach it to the EC2 instance
+resource "aws_ebs_volume" "aws_ebs_volume_terraform" {
+  availability_zone     = "eu-north-1a"
+  size                  = 8 #8 GiB RAM
```

Comment:
This is not a RAM. Check https://ru.wikipedia.org/wiki/%D0%9E%D0%BF%D0%B5%D1%80%D0%B0%D1%82%D0%B8%D0%B2%D0%BD%D0%B0%D1%8F_%D0%BF%D0%B0%D0%BC%D1%8F%D1%82%D1%8C